pyroclast/common/adversarial.py

```diff
@@ -0,0 +1,82 @@
+import tensorflow as tf
+import numpy as np
+
+
+def fast_gradient_sign_method(feature_fn, classify_fn, x, y, eps, norm):
+    """
```

Comment:
Give a description of the method and what it does.

Reply:
Also, talk about how it takes advantage of ReLUs so we don't waste any more time.